### ISSUE TITLE
[论文复现] 动态图的API缺失

### ISSUE THREAD

**Issue by 1340896123** [PaddlePaddle/Paddle#26553]
在pytorch  许多的api接口,.对应paddle的都是静态图的 paddle.fluid.layer  ,但是在论文复现中,paddle使用动态图的方式更好(毕竟调试方便),但是paddle里面缺少了对应api,比如最常见的relu的都没有,所以论文复现只能使用静态图方式,(静态图和动态图也不知道能不能混用,即使混用了,代码维护也极为不便),比如这段pytorch代码
   self.pyramid_fusion_op = nn.Sequential(
            nn.Conv3d(out_dims * 2, 2048, 1, 1, 0, bias=False),
            nn.BatchNorm3d(2048),
            nn.ReLU(inplace=True)
        )
调用
      out = self.fusion_conv(input)

转为paddle代码
  self.fusion_conv = fluid.dygraph.Sequential(
            fluid.dygraph.Conv3D(in_dims, out_channhels, 1, 1, 0),
            fluid.dygraph.BatchNorm(out_channhels),
        )
调用
     out = self.fusion_conv(input)
     out=fluid.layers.relu(out)

这样是不是很奇怪?明明封装好的,非得要东拼西凑才能搞出来,如果这里在用paddle实现pytorch的nn.ModulesList(),这不就是再考验我的耐心吗?

希望官方能在2.0正式版中把动态图API完善

可能上面说的不对,也可能有更好的解决办法,只怪我太菜.,没有找到,也没想到.

**Comment by yongqiangma** [comment]
2.0里面已经有相关的功能，在develop分支的文档已经可以查到
https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api_cn/nn_cn/ReLU_cn.html#relu
